src/aposteriori/dnn/network/visualization/prediction_entropy.py

```diff
@@ -1,44 +1,44 @@
 import urllib
 import typing as t
+import warnings
 from pathlib import Path
 
 import numpy as np
 import ampal
 import tensorflow as tf
-
 from ampal.protein import Polypeptide
 from scipy.stats import entropy
 
 from aposteriori.dnn.config import (
     ANNOTATED_ENTROPY_PDB_PATH,
-    FRAME_EDGE_LENGTH,
+    # FRAME_EDGE_LENGTH,
```

Comment:
Worth removing the commented arguments now?

Reply:
yup, good catch